docs/cli/defradb_client.md

```diff
@@ -1,11 +1,11 @@
 ## defradb client
 
-Interact with a running DefraDB node as a client
+Interact with a DefraDB node
```

Comment:
Nitpick: here we say DefraDB node. Above we say database node in the start command. Should be consistent IMO

Reply:
Alright will change it :)

Reply:
There are some other instances too, lets make all consistent